lib/include/dlisio/lis/io.hpp

```diff
@@ -100,11 +101,16 @@ struct record_index {
      */
     bool is_incomplete() const noexcept (true);
 
+    /* Stores the error msg from the (if applicable) failed indexing process
```

Comment:
It doesn't store the message, it returns it.

Plus: Propegate -> Prop(a)gate in commit message